bucket/arcadedb.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "23.6.1",
+    "description": "Multi-model database, one DBMS that supports SQL, Cypher, Gremlin, HTTP/JSON, MongoDB and Redis.",
+    "homepage": "https://arcadedb.com",
+    "license": "Apache-2.0",
+    "notes": [
+        "After an installation or update please close all your terminal sessions before interacting with ArcadeDB commands.",
+        "Available commands:",
+        "  arcadedb-console",
+        "  arcadedb-server"
+    ],
```

Comment:
```suggestion
```

- With Current junction, seem we does not need to close the terminal session after install/update
- `scoop info <app>` will list commands/executable file/binaries

Reply:
Ok. Good to know! Thanks